CONTRIBUTING.md

```diff
@@ -54,30 +54,21 @@ Run unit testing
 make test-unit
 ```
 
+Before running integration/e2e/scalability testing, deploy the placement controller with corresponding steps from [PLACEMENT Doc](https://github.com/open-cluster-management-io/placement#deploy-the-placement-controller).
```

Comment:
I do not think integration relies on an existing placement controller. We leverage kubebuilder actually.

make test-e2e seems also deploy hub as a pre-step?

Reply:
Thanks for the comments, I misleade the integration part. 

I had updated the fixes, integration part will not be touched, and removed `deploy-hub` for e2e/scalability testing from Makefile to avoid duplicated steps, please help review.

Reply:
hrm, I think it is needed in e2e, not sure about scalability. Since we have a github action to run e2e that will need to run deploy for e2e test

Reply:
Yes, just like you mentioned, we still need the `deploly-hub` for e2e, and I had updated the doc again for this, the related logic will not be removed from. 

The branch for this ticket is broken, I opened #47 for the new fix, please help review in that ticket.